generators/generator-base-private.js

```diff
@@ -887,11 +887,17 @@ module.exports = class JHipsterBasePrivateGenerator extends Generator {
       } else {
         const javaVersion = stderr.match(/(?:java|openjdk) version "(.*)"/)[1];
         if (
+          !javaVersion.match(new RegExp('16'.replace('.', '\\.'))) &&
```

Comment:
Is `.replace('.', '\\.')` required?
```suggestion
          !javaVersion.match(new RegExp('16')) &&
```

Reply:
You're right for > 1.8 it is not required anymore.